src/Java.Interop.Tools.Generator/Metadata/FixupXmlDocument.cs

```diff
@@ -58,9 +58,9 @@ public void Apply (ApiXmlDocument apiDocument, string apiLevelString, int produc
 						else
 							// BG8A00
 							Report.LogCodedWarning (0, Report.WarningRemoveNodeMatchedNoNodes, null, metaitem, $"<remove-node path=\"{path}\" />");
-					} catch (XPathException e) {
+					} catch (XPathException) {
 						// BG4301
-						Report.LogCodedError (Report.ErrorRemoveNodeInvalidXPath, e, metaitem, path);
+						Report.LogCodedError (Report.ErrorRemoveNodeInvalidXPath, metaitem, path);
```

Comment:
Why don't we want to include `e` anymore?

Reply:
We don't use it anywhere, we simply print out the localized message.